### DISCUSSION TITLE
Add Tab to part that present marking code find on components like smd transistors or IC

### DISCUSSION THREAD

**Discussion by Muzeau** [Part-DB/Part-DB-server#199] [Ideas]
How it is possible to add a tab to Part ?

**Comment by jbtronics** [comment]
The marking code is nothing which you automatically could deduce from the part, so you will have to manually supply it for every part.

However you can put the marking code into the comment field of a part or define it in a Part Parameter/Specification (then you can even filter for it quite easily via the filter system). Basically the specification feature allows you to define custom fields for parts and other entities, for which you then can search and filter via the filter system.
